addon/query-cache.js

```diff
@@ -67,6 +67,18 @@ export default class QueryCache {
     delete this._reverseQueryCache[id];
   }
 
+  unloadURL(cacheKey) {
+    if (this.contains(cacheKey)) {
+      delete this._queryCache[cacheKey];
+    }
+  }
+
+  contains(recordOrCacheKey) {
```

Comment:
Agreed. Will do it and add a test.

Reply:
Do you think the `containsURL` name makes sense here, since we're passing a `cacheKey`? Something more accurate would probably be `containsCachedURLQuery(cacheKey)`, but I don't like this either. Thoughts?